configs/Asus/P10S-M_WS.conf

```diff
@@ -0,0 +1,60 @@
+# P10S-M WS
+chip   "nct6791-*" 
+    ignore temp1
+    ignore temp2
+    ignore temp3
+    ignore temp4
+    ignore temp6
+    ignore temp8
+    ignore temp9
+    ignore temp10
+    label temp7 "CPU"
```

Comment:
That's a lot of `ignore`s. Do all of these give useless readings? What are they? Can you add some comments explaining it?

Reply:
Sure, `temp8` to `temp10` gives always 0, `temp1` and `temp6` gives always `-62.0`, `temp2` gives `-45.0`.

```
SYSTIN:        -62.0°C  (high = +90.0°C, hyst = +90.0°C)  sensor = thermistor
CPUTIN:        -45.0°C  (high = +90.0°C, hyst = +91.0°C)  sensor = thermal diode
AUXTIN3:       -62.0°C    sensor = thermistor
PCH_CHIP_CPU_MAX_TEMP:   +0.0°C  
PCH_CHIP_TEMP:           +0.0°C  
PCH_CPU_TEMP:            +0.0°C
```
Regarding `temp3` and `temp4` I am not 100% sure, but it seems like it's always same.
```
AUXTIN0:       +23.0°C    sensor = thermistor
AUXTIN1:       +26.0°C    sensor = thermistor
```

Should I add this as comments into PR ?

Reply:
> Should I add this as comments into PR ?

Yes, please. Into the config itself.